unleash/src/main/java/com/silvercar/unleash/util/UnleashConfigBuilder.kt

```diff
@@ -53,6 +60,7 @@ fun unleashConfig(func: UnleashConfigBuilder.() -> Unit) = UnleashConfigBuilder(
 
   fun customHttpHeadersProvider(provider: CustomHttpHeadersProvider): UnleashConfigBuilder {
     customHttpHeadersProvider = provider
+//    customHttpHeadersProvider = provider
```

Comment:
Can this line be deletetd?

Reply:
Good catch, that shouldn't be there. I'll push a fix later today.